pages/reports.vue

```diff
@@ -0,0 +1,153 @@
+<template>
```

Comment:
@pjvv Does index.vue want to be removed and add a redirect from root index to /reports?

Reply:
Yeah, I'm going to try do that today.

Reply:
I thought about leaving the page as a "Dashboard" we can build out with some nerdy stats as we go along. Boxes scanned, samples processed, files parsed etc.